examples/postgresql/model.js

```diff
@@ -69,12 +70,14 @@ model.getClient = function (clientId, clientSecret, callback) {
 model.getRefreshToken = function (bearerToken, callback) {
   pg.connect(connString, function (err, client, done) {
     if (err) return callback(err);
-    client.query('SELECT refresh_token, client_id, expires, user_id FROM oauth_refresh_tokens ' +
-        'WHERE refresh_token = $1', [bearerToken], function (err, result) {
-      // The returned user_id will be exposed in req.user.id
-      callback(err, result.rowCount ? result.rows[0] : false);
-      done();
-    });
+	// Note: To avoid replicating the scope string in both token tables, the old
+	// access token's scope string must be retrieved and passed along from here.
+	client.query('SELECT rt.refresh_token, rt.client_id, rt.expires, rt.user_id, at.scope FROM ' +
```

Comment:
This assumes that expired `accessToken`'s are not deleted. This takes away control from the developer since he'll be forced to keep all of the access tokens around.

I see two possible solutions for this:

1. We can add the `scopes` column to the `oauth_refresh_tokens` table.
2. We can merge the two tables into one: `oauth_access_tokens` and `oauth_refresh_tokens` would become `oauth_tokens` with a `type` column. There really isn't any difference between these tables right now.

@thomseddon Maybe I'm missing something. What is the reason for having two tables right now? Is it for the purpose of this example or is it recommended by the spec?